pkgs/development/compilers/crystal/default.nix

```
@@ -97,33 +99,38 @@ let
 
     # This makes sure we don't keep depending on the previous version of
     # crystal used to build this one.
-    CRYSTAL_LIBRARY_PATH = "${placeholder "out"}/lib/crystal";
+    CRYSTAL_LIBRARY_PATH = "${placeholder "lib"}/crystal";
 
     # We *have* to add `which` to the PATH or crystal is unable to build stuff
     # later if which is not available.
     installPhase = ''
       runHook preInstall
 
-      install -Dm755 .build/crystal $out/bin/crystal
-      wrapProgram $out/bin/crystal \
+      install -Dm755 .build/crystal $bin/bin/crystal
```

Isn't it?
```suggestion
      install -Dm755 .build/crystal $bin/crystal
```
(ditto below)